examples/cloud/temporal/README.md

```
@@ -0,0 +1,209 @@
+1. Open `http://localhost:8233` in your browser
+2. Navigate to the "Workflows" section
+3. You'll see a list of all workflow executions, their status, and other details
+4. Click on a workflow to see its details, history, and to send signals
+
+## Deploy to mcp-agent Cloud
+
+You can deploy this MCP-Agent app as a hosted mcp-agent app in the Cloud.
+
+1. In your terminal, authenticate into mcp-agent cloud by running:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)